src/bindings/c/include/openvino/c/ov_node.h

```diff
@@ -17,12 +17,21 @@
 typedef struct ov_output_const_node ov_output_const_node_t;
 typedef struct ov_output_node ov_output_node_t;
 
+/**
+ * @struct ov_output_const_node_list_t
+ * @brief Represents an array of const ov_output_nodes.
+ */
+typedef struct {
+    ov_output_const_node_t* output_ports;
+    size_t size;
+} ov_output_const_node_list_t;
```

Comment:
I see that it isn't a hidden structure.
So my proposal with any name can look like:
```
CHECK_STATUS(ov_output_node_get_any_name(input_ports.output_ports[0], &input_tensor_name));
```

Reply:
`input_ports.output_ports` is an opaque handle, we cannot use its index to access it.